common/src/main/java/com/hedera/block/common/utils/Preconditions.java

```diff
@@ -107,39 +112,36 @@ public static int requirePositive(final int toCheck, final String errorMessage)
      * @param toCheck the long to check if it is a positive power of two
      * @return the long to check if it is positive
      * @throws IllegalArgumentException if the input long to check is not
-     *                                  positive
+     * positive
      */
     public static long requirePositive(final long toCheck) {
-        return requirePositive(toCheck, null);
+        return requirePositive(toCheck, DEFAULT_REQUIRE_POSITIVE_MESSAGE);
     }
 
     /**
      * This method asserts a given long is a positive. A long is positive
      * if it is NOT equal to zero and is greater than zero.
      *
-     * @param toCheck      the long to check if it is a positive power of two
-     * @param errorMessage the error message to be used in the exception if the
-     *                     input long to check is not positive, if null, a default
-     *                     message will
-     *                     be used
-     * @return the long to check if it is positive
+     * @param toCheck the long to check if it is a positive power of two
+     * @param errorMessage a formatted string with one decimal parameters for
+     * {@code toCheck}, must not be {@code null}.<br/>
+     * Example error message: {@code "The input number (%d) must be positive."}
```

Comment:
These example items might be easier if you use `@value` tags.
```suggestion
     * Example error message: {@value #DEFAULT_REQUIRE_POSITIVE_MESSAGE}
```

Reply:
✅ Thanks for that, indeed this makes the javadoc better and if ever the default message changes, the javadoc will reflect that. I've made this improvement as well!